packages/utilities/psammead-storybook-helpers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-storybook-helpers",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Can this be `2.1.0` please as it ['adds functionality in a backwards-compatible manner'](https://semver.org/) :)